src/shortcuts/cursorDown.js

```diff
@@ -1,6 +1,7 @@
 import { store } from '../store.js'
 
 // util
+import React from 'react'
```

Comment:
This import should go at the top of the file. "util" is reserved for imports from `util.js`.

Reply:
Sorry about that,  I must have done that hurriedly. I'll be more careful next time.